language/move-compiler/src/parser/comments.rs

```diff
@@ -34,3 +34,20 @@ pub fn verify_string(file_hash: FileHash, string: &str) -> Result<(), Diagnostic
         }
     }
 }
+
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn test_verify_string_error_escapes_unicode() {
+        let checkmark = "✓";
+        let result = super::verify_string(
+            super::FileHash::new("unused"),
+            checkmark,
+            );
+        let diagnostics = result.unwrap_err();
+        let err_msg = format!("{:?}", diagnostics);
+        assert!(err_msg.contains(&checkmark.escape_default().to_string()));
+        assert!(!err_msg.contains(&checkmark));
+    }
+}
```

Comment:
Could you add a test for this instead in `language/move-compiler/tests/move_check/parser/`

Reply:
Test moved, and rebased into this commit.